driver/pace/driver/driver.py

```diff
@@ -185,49 +189,50 @@ def __init__(
         """
         logger.info("initializing driver")
         self.config = config
+        self.time = self.config.start_time
         self.comm_config = config.comm_config
         self.comm = config.comm_config.get_comm()
         self.performance_config = self.config.performance_config
         with self.performance_config.total_timer.clock("initialization"):
             communicator = pace.util.CubedSphereCommunicator.from_layout(
                 comm=self.comm, layout=self.config.layout
             )
-            quantity_factory, stencil_factory = _setup_factories(
+            self.quantity_factory, self.stencil_factory = _setup_factories(
```

Comment:
I don't think these two are used in call time? Is it more of a consistent style choice?

Reply:
I wanted these to become public API so it's easier to interact with a `driver` class externally, e.g. to initialize a d2a2c object that matches the settings of the driver and pass the driver's state into it (which I do in the demo notebook).